tests/virtualization/xen/dom_metrics.pm

```diff
@@ -0,0 +1,30 @@
+# XEN regression tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Test if Dom0 metrics are visible to VM
+# Maintainer: Jan Baier <jbaier@suse.cz>
+
+use base 'xen';
+
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    ## TODO:
+    #    * start some systems in xl stack
```

Comment:
I'm not sure if we should rush in creating such test modules, do you plan to work on them in this PR or in following ones?

Reply:
I plan to gift that part to another QAM engineer :smiley: